### ISSUE TITLE
Make icons accessible in different platforms.

### ISSUE THREAD

**Issue by AshutoshDash1999** [Bookingjini-Labs/bookingjini-icons#219]
### Is your feature request related to a problem? Please describe.

Currently, the icons are accessible only for HTML pages. But we want it to be accessible to different platforms so that it can be used in React, React Native etc. 

### Describe the solution you'd like

Please mention if you want to work on this and also let us know for which platform you can do this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kartikjoshi267** [comment]
@AshutoshDash1999 I wanted to work on this issue for React. But, I have a query!
Do you want to create an NPM package for these icons or you just want me to put the \<i\> tags with React attributes like `className`?

**Comment by AshutoshDash1999** [comment]
@kartikjoshi267 Please create an NPM package

**Comment by kartikjoshi267** [comment]
> @kartikjoshi267 Please create an NPM package

I can try! Can you please assign this issue to me?

**Comment by AshutoshDash1999** [comment]
@kartikjoshi267 assigned this to you

**Comment by kartikjoshi267** [comment]
@AshutoshDash1999 Shall I create a new branch for the NPM package named as package and also, add some documentation for it?
Also, I will not publish the package, so please publish the package on your own!

**Comment by AshutoshDash1999** [comment]
yes, please create a fork and create a seperate folder to keep all the package related files

**Comment by kartikjoshi267** [comment]
@AshutoshDash1999  Shall I publish the package or you will do it on your own??

**Comment by AshutoshDash1999** [comment]
we will publish the package of our own

**Comment by AshutoshDash1999** [comment]
@kartikjoshi267 have you published the package of your own?

**Comment by kartikjoshi267** [comment]
> @kartikjoshi267 have you published the package of your own?

No, I haven't done it

**Comment by AshutoshDash1999** [comment]
@kartikjoshi267 can you add typescript support to it?

**Comment by kartikjoshi267** [comment]
> @kartikjoshi267 can you add typescript support to it?
> 

Actually I don't have any idea of typescript so can't help!

**Comment by 0xabdulkhaliq** [comment]
@AshutoshDash1999 

Assign this issue to me, I can make this work Done!

Thank You.
